docs/content/quickstart.md

```diff
@@ -38,6 +30,174 @@ helm installation to perform all steps it needs is to edit the
 Assuming `tiller` is deployed in your `kube-system` namespace, this will
 give it very open permissions.
 
-### Installation Examples
+## Installing Maesh
+
+```bash tab="Command"
```

Comment:
If there's a command tab, I guess there should be an expected output as well?

Reply:
Way too long, isn't it ?

Reply:
You could also do a `[...]` truncated output, just to show the top and bottom perhaps...

Reply:
Yep okay.